integrations/premake.rst

```diff
@@ -1,32 +1,103 @@
+.. _premake:
+
+premake
+=======
 
 |premake_logo| Premake
 _________________________
 
-`Premake`_ version 4 has **experimental** support as a generator package.
+Since conan 1.9.0, premake generator is built-in, so the following should be enough to use it:
+
+.. code-block:: text
+
+    [generators]
+    premake
+
+Example
+-------
+
+In order to use new generator within your project, use the following as a reference:
+
+.. code-block:: lua
+
+   -- premake5.lua
+
+   require 'conanbuildinfo'
+
+   workspace "ConanPremakeDemo"
+     configurations { "Debug", "Release" }
+     platforms { "Win32", "x64" }
+
+     filter { "platforms:Win32" }
+      system "Windows"
+      architecture "x32"
+
+     filter { "platforms:x64" }
+      system "Windows"
+      architecture "x64"
+
+   project "ConanPremakeDemo"
+     kind "ConsoleApp"
+     language "C++"
+     targetdir "bin/%{cfg.buildcfg}"
+
+     includedirs { conan_includedirs }
+     libdirs { conan_libdirs }
+     links { conan_libs }
+     linkoptions { conan_exelinkflags }
+
+     files { "**.h", "**.cpp" }
 
-You can find this generator in this repository: https://github.com/memsharded/conan-premake
+     filter "configurations:Debug"
+      defines { "DEBUG" }
+      symbols "On"
 
-In order to use it, clone the repository and export the recipe to the local cache:
+     filter "configurations:Release"
+      defines { "NDEBUG" }
+      optimize "On"
+
+Demo requires two remotes to run: ``bincrafters`` and ``conan-community``. Add them, if necessary:
 
 .. code-block:: bash
 
-    $ git clone https://github.com/memsharded/conan-premake
-    $ conan export conan-premake memsharded/testing
+    $ conan remote add conan-community https://api.bintray.com/conan/conan-community/conan
+    $ conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
 
-Now you can use this generator as a requirement in your recipes **but also as a generator**:
+Now let's get the premake project and build it:
 
-.. code-block:: text
+.. code-block:: bash
 
-    [requires]
-    PremakeGen@0.1@memsharded/testing
+    $ git clone https://github.com/SSE4/conan-premake-demo.git
```

Comment:
Having a look at the project and the need to add two addtional remotes to run the example, I think it is too much for a premake example.

- First: If a user wants to use premake, it will have premake already installed, so no need to add premake_installer as build require
- Second: Conan community should not be added as a repository, it only exists for maturing packages
- Third: Avoid using scripts like `./run.cmd`, is better to use eveything explicitly and with clear steps